### PR TITLE
Adding OpenID Connect Instructions for SSO

### PR DESCRIPTION
**Pull request by ryanrath**
## Description
Added a new documentation page that describes the steps that need to be taken to setup OpenID Connect as an Identity Provider for XDMoD. 

## Motivation and Context
We had a support ticket asking for how to set this up and realized that while we use OpenID Connect as an IDP in one of our instances, we didn't have any documentation for how to set it up. This rectifies that problem.

## Checklist:

- [X] The pull request description is suitable for a Changelog entry
- [X] The milestone is set correctly on the pull request
- [X] The appropriate labels have been added to the pull request

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -38,16 +38,6 @@ jobs:
...
-      - restore_cache:
-          keys:
-            # "composer.lock" can be used if it is committed to the repo
-            - v1-el8-dependencies-{{ checksum "composer.json" }}
-            # fallback to using the latest cache if no exact match is found
-            - v1-el8-dependencies-
-      - save_cache:
-          key: v1-el8-dependencies-{{ checksum "composer.json" }}
-          paths:
-            - ./vendor
```
Comment (eiffel777) [inline .circleci/config.yml:50]: If this is just a documentation PR why is this being removed? If this needs to be removed, should it be done in a different PR?

Comment (ryanrath) [reply .circleci/config.yml:50]: Needed to make it so that the ci would run.

**Review by eiffel777 — APPROVED** [review]
